codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHTTPBindingProtocolGenerator.kt

```diff
@@ -53,8 +53,6 @@ abstract class AWSHTTPBindingProtocolGenerator(
             protocolTestTagsToIgnore,
         ).generateProtocolTests() + renderEndpointsTests(ctx)
 
-    override fun generateSmokeTests(ctx: ProtocolGenerator.GenerationContext) = AWSSmokeTestGenerator(ctx).generateSmokeTests()
-
```

Comment:
This override is no longer needed since the smoke test generator provided by the customizer is now always used.